webapp/src/Service/ScoreboardService.php

```diff
@@ -115,22 +115,17 @@ public function getScoreboard(
      * Get scoreboard data for a single team based on the cached data in the
      * scorecache table.
      *
-     * @param Contest $contest The contest to get the scoreboard for.
-     * @param int     $teamId  The ID of the team to get the scoreboard for.
-     * @param bool    $jury    If true, the scoreboard will always be current.
-     *                         If false, frozen results will not be returned.
+     * @param Contest $contest         The contest to get the scoreboard for.
+     * @param int     $teamId          The ID of the team to get the scoreboard for.
+     * @param bool    $showFtsInFreeze If false, the scoreboard will hide first
+     *                                 to solve for submissions after contest freeze.
      * @return Scoreboard|null
      * @throws \Exception
      */
-    public function getTeamScoreboard(Contest $contest, int $teamId, bool $jury = false)
+    public function getTeamScoreboard(Contest $contest, int $teamId, bool $showFtsInFreeze = true)
     {
         $freezeData = new FreezeData($contest);
 
-        // Don't leak information before start of contest
-        if (!$jury && !$freezeData->started()) {
-            return null;
-        }
-
```

Comment:
Why are you removing this code? Was it irrelevant to begin with?

Reply:
Jury is always true. At least I could not find any place in our code where it was false.
So this if always evaluated to false...